helm/examples/tracing/README.md

```diff
@@ -0,0 +1,94 @@
+# HPCC Component Instrumentation and Tracing
+
+The HPCC Platform is instrumented to optionally track distributed work actions (Traces) and their various sub-task (spans) via Open Telemetry tooling. Traces and spans track information vital information which can be reported to services which specialize in processing and visualization of standardized trace/span information for the purpose or monitoring and observability of the health of individual requests processed by the platform.
+
+Tracing of HPCC components is enabled by default, and can be configured to fit the needs of the given deployment.
+
+## Configuration
+All configuration options detailed here are part of the HPCC Systems Helm chart, and apply at the global or component level.
+
+### Tracing cofiguration options
+- disabled - (default: false) disables tracking and reporting of internal traces and spans
+- alwaysCreateGlobalIds - If true, assign newly created global ID to any requests that do not supply one.
+- exporter - Defines The type of exporter in charge of forwarding span data to target back-end
+ - type - (defalt: NONE) "OTLP-HTTP" | "OTLP-GRCP" | "OS" | "NONE"
+  - OTLP-HTTP
+   - url - (default localhost:4318) The url of the target OTLP-HTTP collector
+   - timeOutSecs - (default 10secs)
+   - consoleDebug - (default false)
+  - OTLP-GRCP
+   - endpoint: (default localhost:4317)     #exporter specific key/value pairs
+   - useSslCredentials
+   - sslCredentialsCACcert
+- processor - Controls span processing style. One by one as available, or in batches.
+ - type  - (default: simple) "simple" | "batch"
+
+### Sample configuration
+Below is a sample helm values block directing the HPCC tracing framework to process span information serially, and export the data over OTLP/HTTP protocol to localhost:4318 and output export debug information to console:
+
+```console
+global:
+  tracing:
+    exporter:
+      type: OTLP-HTTP
+      consoleDebug: true
+    processor:
+      type: simple
+```
+### Sample configuration command
+
+Sample helm command deploying an HPCC chart named myTracedHPCC using the hpcc helm repo and providing a the above tracing configuration.
+
+```console
+helm install myTracedHPCC hpcc/hpcc -f otlp-http-collector-default.yaml
+```
+## Tracing information
+HPCC tracing information includes data needed to trace requests as they traverse over distributed components and information related to sub-tasks actuated to process the request in the form of spans. Each trace and all its related spans are assigned unique IDs which follow the Open Telemetry standard.
```

Comment:
Is it possible to reword "information related to sub-tasks actuated to process the request in the form of spans".  I found it hard to understand, especially actuated.